include/dpp/channel.h

```diff
@@ -383,23 +383,29 @@ class DPP_EXPORT channel : public managed, public json_interface<channel>  {
 	std::string get_mention() const;
 
 	/**
-	 * @brief Get the user overall permissions for a member on this channel, including channel overwrites.
+	 * @brief Get the overall permissions for a member in this channel, including channel overwrites, role permissions and admin privileges.
 	 * 
 	 * @param user The user to resolve the permissions for
-	 * @return permission Permissions bitmask made of bits in dpp::permissions.
+	 * @return The permission overwrites for the member. Made of bits in dpp::permissions.
```

Comment:
the word permission on the start here is a type. vscode puts it here and doxygen doesn't support it yet, but it might some day soon...

Reply:
oh okay. ill change it

Reply:
Changed